eng/intellisense.targets

```diff
@@ -0,0 +1,32 @@
+<Project>
+
+  <PropertyGroup>
+      <!-- Search for the documentation file in the intellisense package and otherwise pick up the generated one. -->
```

Comment:
nit: indentation is off

```suggestion
    <!-- Search for the documentation file in the intellisense package and otherwise pick up the generated one. -->
```